pkg/registry/core/service/ipallocator/allocator.go

```diff
@@ -262,11 +262,14 @@ func calculateIPOffset(base *big.Int, ip net.IP) int {
 // RangeSize returns the size of a range in valid addresses.
 func RangeSize(subnet *net.IPNet) int64 {
 	ones, bits := subnet.Mask.Size()
-	if bits == 32 && (bits-ones) >= 31 || bits == 128 && (bits-ones) >= 63 {
+	if bits == 32 && (bits-ones) >= 31 || bits == 128 && (bits-ones) >= 121 {
```

Comment:
This could use a comment

Reply:
I will add a comment.

Reply:
Or maybe it's better to make it comparable with IPv4, so it won't be confusing. I'll change the 121 to 127